flake.nix

```diff
@@ -15,95 +15,101 @@
   };
 
   outputs = { self, nixpkgs, flake-utils, flakebox, advisory-db }:
+    let
+      overlays = [
+        (final: prev: {
+
+          wasm-bindgen-cli = final.rustPlatform.buildRustPackage rec {
+            pname = "wasm-bindgen-cli";
+            version = "0.2.89";
+            hash = "sha256-IPxP68xtNSpwJjV2yNMeepAS0anzGl02hYlSTvPocz8=";
+            cargoHash = "sha256-pBeQaG6i65uJrJptZQLuIaCb/WCQMhba1Z1OhYqA8Zc=";
+
+            src = final.fetchCrate {
+              inherit pname version hash;
+            };
+
+            nativeBuildInputs = [ final.pkg-config ];
+
+            buildInputs = [ final.openssl ] ++ final.lib.optionals final.stdenv.isDarwin [
+              final.curl
+              final.darwin.apple_sdk.frameworks.Security
+            ];
+
+            nativeCheckInputs = [ final.nodejs ];
+
+            # tests require it to be ran in the wasm-bindgen monorepo
+            doCheck = false;
+          };
+
+          rocksdb_7_10 = prev.rocksdb_7_10.overrideAttrs (oldAttrs:
+            final.lib.optionalAttrs final.stdenv.isDarwin {
+              # C++ and its damn super-fragie compilation
+              env = oldAttrs.env // {
+                NIX_CFLAGS_COMPILE = oldAttrs.env.NIX_CFLAGS_COMPILE + " -Wno-error=unused-but-set-variable";
+              };
+            });
+
+          rocksdb_6_23 = prev.rocksdb_6_23.overrideAttrs (oldAttrs:
+            final.lib.optionalAttrs final.stdenv.isDarwin {
+              # C++ and its damn super-fragie compilation
+              env = oldAttrs.env // {
+                NIX_CFLAGS_COMPILE = oldAttrs.env.NIX_CFLAGS_COMPILE + " -Wno-error=unused-but-set-variable -Wno-error=deprecated-copy";
+              };
+            });
```

Comment:
The old rocksdb version is only used by electrs which we only compile once and then cache anyway. Why do we provide rocksdb as a linked lib instead of just letting the crate compile it in (which is the default if no external library is defined)?